src/components/sbb-teaser-hero/sbb-teaser-hero.tsx

```diff
@@ -109,15 +109,15 @@ export class SbbTeaserHero {
     return (
       <a class="teaser-hero" href={this.link} {...linkAttributes}>
         <sbb-image
+          aspect-ratio="1-1"
+          no-radius="true"
```

Comment:
does the sbb teaser hero always have no border radius? because then it could be coupled to `variant-teaser-hero`

Reply:
aspect ratio 1-1 was a default in sbb-image. but not any more. If we want 1-1, we need to set it.
same for no-radius: radius is yet default, if we don't want, we need to specify it.

Reply:
Here I meant the radius, so if every teaser hero should have an image with no border radius, then it does not have to be set here, but could be covered with the variant teaser. To be more clear: If your image component detects that it is in variant teaser, then it remove the radius, or sets the aspect ratio (as already done).

Reply:
I prefer to not add more specific logic in the sbb-image. 
- There is no choice for the aspect-ratio (special case that we should do other way - like aspect-ratio config which is working like pictureSizesConfig -  but would be overkill for now), 
- but the radius is a simple attribute.. if the user of the sbb-image (here the teaser-hero) want it or not, just use the attribute.

Reply:
fixed

Reply:
thanks! :)